elements/lisk-p2p/test/unit/peer_discovery.ts

```diff
@@ -65,6 +66,10 @@ describe('peer discovery', () => {
 				});
 			});
 
+			afterEach(async () => {
+				sandbox.restore();
```

Comment:
it's restore `afterEach` in global level

Reply:
Global restore doesn't apply over this test that's why I've added it.

Reply:
can you check why it's not applied here...? maybe config is wrong